pkgs/development/python-modules/httpx/default.nix

```diff
@@ -1,71 +1,61 @@
 { lib
 , buildPythonPackage
+, pythonOlder
 , fetchFromGitHub
-, fetchpatch
+, brotli
 , certifi
-, chardet
-, h11
 , h2
 , httpcore
-, idna
 , rfc3986
 , sniffio
-, isPy27
-, pytest
+, pytestCheckHook
 , pytest-asyncio
 , pytest-trio
 , pytestcov
 , trustme
 , uvicorn
-, brotli
 }:
 
 buildPythonPackage rec {
   pname = "httpx";
-  version = "0.14.2";
-  disabled = isPy27;
+  version = "0.16.1";
+  disabled = pythonOlder "3.6";
 
```

Comment:
looks like a regression:

```
builder for '/nix/store/667gj5w9jcysm7pad0b4lma5hv4zkggf-python3.7-asgi-csrf-0.7.drv' failed with exit code 2; last 10 log lines:
  ImportError while importing test module '/build/source/test_asgi_csrf.py'.
  Hint: make sure your test modules/packages have valid Python names.
  Traceback:
  test_asgi_csrf.py:7: in <module>
      from httpx._content_streams import MultipartStream
  E   ModuleNotFoundError: No module named 'httpx._content_streams'
  =========================== short test summary info ============================
  ERROR test_asgi_csrf.py
  !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
  =============================== 1 error in 0.15s ===============================
cannot build derivation '/nix/store/hda4799m3wcb5mfazk2x1s77i0v8qykr-python3.7-datasette-0.46.drv': 1 dependencies couldn't be built
```

Reply:
Probably related to https://github.com/encode/httpx/commit/354c4cac1f1796ee4d8861a4c978807398852ad5.

Reply:
Reported in https://github.com/simonw/asgi-csrf/issues/18.

Reply:
looks like they imported a private module... so shame on them

Reply:
maybe just disable that test?

Doesn't look like it's failing on anything exported by the module

Reply:
Disabled the tests on asgi-csrf and bumped the package to the latest version for good measure.